code/modules/organs/limbs.dm

```diff
@@ -1074,6 +1079,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 		"[text1]",
 		"[text2]")
 		add_limb_flags(LIMB_SPLINTED)
+		splint_stacks = max(splint_stacks, applied_stacks)
```

Comment:
Why the max(), you already check that in line 1056?

Reply:
removed max